docs/_static/nidigital_usage.inc

```diff
@@ -1,27 +1,45 @@
 Usage
 ------
 
-The following is a basic example of using the **nidigital** module to open a session to a ....
+The following is a basic example of using the **nidigital** module to open a session to a digital pattern instrument
+and source/measure both voltage and current using the PPMU on selected channels.
```

Comment:
Only current is being sourced. So, please change to:
"open a session to a digital pattern instrument, source current, and measure both voltage and current using the PPMU on selected channels."

Reply:
Done